supplements/eberron-rising-from-the-last-war/races/orc.xml

```diff
@@ -33,15 +33,15 @@
 				<tr><td>7</td><td>The leader of your clan committed a great wrong against another clan, which has now sworn to exterminate every last member of your clan. What better time to see the world?</td></tr>
 				<tr><td>8</td><td>Your work as an investigator has set you upon a trail of mysteries that leads far beyond the Marches.</td></tr>
 			</table>
-			<h4>HALF-0RCS OF KHORVAIRE</h4>
+			<h4>HALF-ORCS OF KHORVAIRE</h4>
 			<p>With the rise of the dragonmarked House Tharashk, orcs and half-orcs have spread through the Five Nations and beyond. In general, the people of the Five Nations know little about half-orcs. When people imagine a halforc, they usually think of a Tharashk bounty hunter or dogged inquisitive, but a few ignorant sorts might still hold the stereotype of a "simpleton from the swamps."</p>
-			<h4>HALF-0RCS AND DRAGONMARKS</h4>
+			<h4>HALF-ORCS AND DRAGONMARKS</h4>
 			<p>The Mark of Finding appears among half-orcs of House Tharashk. Strangely, both orcs and humans associated with the house cannot develop this mark. As a result, dragonmarked half-orcs work extensively with humans, orcs, and unmarked half-orcs in pursuit of the house's mission of bounty hunting, inquisitive work, and prospecting. Across Khorvaire, anyone who wants to find something or someone turns to House Tharashk.</p>
 			<h4>ORC TRAITS</h4>
 			<p>An orc character has the following racial traits.</p>
 			<p class="indent"><strong><em>Ability Score Increase.</em></strong> Your Strength score increases by 2, and your Constitution score increases by 1.</p>
-			<p class="indent"><strong><em>Age.</em></strong> Ores reach adulthood at age 12 and live up to 50 years .</p>
-			<p class="indent"><strong><em>Alignment.</em></strong> The ores of Eberron are a passionate people, given to powerful emotion and deep faith. They are generally chaotic, but can be any alignment.</p>
+			<p class="indent"><strong><em>Age.</em></strong> Orcs reach adulthood at age 12 and live up to 50 years .</p>
```

Comment:
It looks like there is also an unneeded space before the period on Age

Reply:
Didn't catch that the first time through - updated in next commit.